### DISCUSSION TITLE
Auto importing from template library locations

### DISCUSSION THREAD

**Discussion by gone** [django-components/django-components#119] [Ideas]
Right now if I have a file that contains a set of components, something like `util/buttons.py`, even though that file is registered in the 'libraries' setting that file needs to be imported to ensure the components are available. I don't think this makes sense - I think the files in libraries should be imported at application boot, so that they're guaranteed to be available.

**Comment by gone** [comment]
It already works this way and I just declared the setting value twice, durp.

**Comment by EmilStenstrom** [comment]
Nice to hear you solved it!
